tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "declaration": true,
+    "lib": ["ESNext"],
+    "module": "commonjs",
+    "target": "es6"
```

Comment:
1. I'm not sure whether `es6` is a good target for `ESNext` `lib`. Any thoughts?
2. add `importHelpers: true` to minimize the build size.
3. add `strict: true` to make typing safer.